app/models/user.rb

```diff
@@ -751,6 +751,19 @@ def can_update_personell?(lecture)
     return false
   end
 
+  # make sure that no IP adress data are stored via the :trackable module
+  # of Devise
+  # see https://github.com/heartcombo/devise/issues/4849#issuecomment-534733131
+
+  def current_sign_in_ip
+  end
+
+  def last_sign_in_ip=(_ip)
+  end
+
+  def current_sign_in_ip=(_ip)
+  end
+
```

Comment:
I extended the comment as suggested. I stayed with the two line empty methods since I do not like the corresponding [recommendation](https://rubystyle.guide/#no-single-line-methods) in the Ruby Style Guide. In a later PR, I will propose a change of the corresponding Rubocop setting.